apps/site/assets/ts/helpers/__tests__/use-line-suspensions-test.ts

```diff
@@ -0,0 +1,8 @@
+import currentLineSuspensions from "../line-suspensions";
+
+// TODO: Add tests
```

Comment:
not blocking but could be worth a follow up ticket to clean up the TODOs around here if we're trying to push this out today